src/components/cms/sections/CalendarSection.tsx

```
@@ -1138,56 +1269,111 @@ export default function CalendarSection({
-                value={description}
-                onChange={(e) => onUpdate?.({ description: e.target.value })}
-                className="w-full p-2 border border-gray-300 rounded-md focus:ring-2 focus:ring-blue-500"
-                rows={2}
-                placeholder="Select from our available services and book your appointment in just a few simple steps."
-              />
+          </div>
+          
+          <div className="flex items-start space-x-2">
+            <FormInput className="h-5 w-5 mt-1 text-muted-foreground" />
+            <div className="flex-1">
+              <div className="isolate" onClick={(e) => e.stopPropagation()}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)